roles/cluster_infra/templates/resources.tf.j2

```diff
@@ -255,8 +264,12 @@ resource "openstack_compute_instance_v2" "login" {
   user_data = <<-EOF
     #cloud-config
     ssh_authorized_keys:
-      - {{ cluster_deploy_ssh_public_key }}
-      - {{ cluster_user_ssh_public_key }}
+    {%- if cluster_ssh_private_key_file is not defined %}
+      - "${openstack_compute_keypair_v2.cluster_keypair.public_key}"
+    {%- endif %}
+    {%- for ssh_key in cluster_deploy_ssh_keys %}
+      - {{ ssh_key }}
+    {%- endfor %}
```

Comment:
The way this is set up now, I don't think `cluster_user_ssh_public_key` will ever make it in right?

Reply:
It wouldn't - I wanted to separate concerns of *deploy* and *user* keys, but perhaps that isn't for now.

Included again.